### DISCUSSION TITLE
v0.0.7

### DISCUSSION THREAD

**Discussion by jk-oster** [jk-oster/obsidian-search-for-web#4] [General]
This release fixes the options page .css and the bug that no results are displayed if the "exluded paths" setting is empty

<hr /><em>This discussion was created from the release <a href='https://github.com/jk-oster/obsidian-search-for-web/releases/tag/'>v0.0.7</a>.</em>

**Comment by marcus-at-localhost** [comment]
Hey, that works now. Thank you!

The Obsidian Search Overlay might need a z-index:
![image](https://user-images.githubusercontent.com/1337223/211496552-05b687f2-05b5-414b-b5ac-8581f19a26d6.png)

And some google icons (in the search bar, but also on youtube) are smaller. I assume it's a side effect of injecting tailwind?

**Reply by jk-oster** [reply]
Thanks for the feedback. It only appears to happen to me when logged in on google, without login it works. I ll adjust the z-index.

Yeah, the icons changed probably due to tailwind styles, maybe I ll switch to classic styling with SCSS to prevent that or find a soultion to limit to which elements the tailwind styles apply.
